plugins/hls-explicit-imports-plugin/src/Ide/Plugin/ExplicitImports.hs

```diff
@@ -494,5 +497,7 @@ constructImport (L lim imd@ImportDecl{ideclName = L _ _, ideclHiding = Just (hid
           containsAvail name avail =
             any (\an -> printOutputable an == (printOutputable . ieName . unLoc $ name))
               $ availNamesWithSelectors avail
-
-constructImport lim _ = lim
+          noExplicitList mList = case mList of
+                                      Nothing      -> True
+                                      Just (_, xs) -> not $ null xs
```

Comment:
This is a super weird case. This is handling `import Foo qualified ()`, right? I guess we might as well handle it, but I think it would be also reasonable to treat that as an explicitly specified import list, even if it's empty.